tools/lkl/tests/lklfuse.sh

```diff
@@ -8,6 +8,7 @@ cleanup()
 {
     set -e
 
+    # TODO remove this sleep?
```

Comment:
Maybe: add an issue to track this.

Reply:
It runs fine for me without the sleep so I'll add an extra commit to drop it.